bootstrap.sh

```diff
@@ -1,26 +1,61 @@
 #!/bin/sh
 
+function pause {
+ 	read -s -n 1 -p "Press any key to continue . . ."
+ 	echo ""
+}
+
+function success {
+	echo Bootstrap succeeded \!
+	exit 0
+}
+
+function error {
+	echo Bootstrap failed \!
+	pause
```

Comment:
pausing is bad, most of those scripts need to run on a CI and it cannot press a key :)

Reply:
In the bootstrap file for windows there is a pause, so I translate it in Bash

Reply:
it shouldn't be in the windows bootstrap either actually

Reply:
Done in 5fe9f31d78ff96b443049d213bbf843715f1a072